web/client/components/widgets/enhancers/wfsTable/common.js

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2018, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/**
+ * Gets the layer URL for the layer
+ * @param {Object} layer
+ * @returns {string} layer url
+ */
+const getLayerUrl = (l = {}) => l.search && l.search.url || l.url;
```

Comment:
This looks more like a selector than an enhancer

Reply:
yep, was common so I put it there. We have very few local selectors, so maybe the best place is `LayerUtils`